### DISCUSSION TITLE
Another cosmological bacground: the CνB

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#107] [Astrophysics and Physics]
"What Exactly Is The Cosmic Neutrino Background? Here's Why It Matters"
[https://youtu.be/KWT01fmXht8](https://youtu.be/KWT01fmXht8)

Anton speculates about future experiments, but these statements are interesting:

*"[... if proven correct, would one day, definitely show us that the Big Bang actually happened](https://youtu.be/KWT01fmXht8?t=5), because we don't have anything convincing right now? 😄 

*"[... but anyway, this was a pretty definitive proof of the Big Bang theory but there were still some inconsistencies](https://youtu.be/KWT01fmXht8?t=157)"*, so not that definitive after all...

**Comment by MJoseph315** [comment]
Wikipedia says there is strong indirect evidence for the existence of the Neutrino Background (https://en.wikipedia.org/wiki/Cosmic_neutrino_background). Two papers cited by this Wikipedia article are this one:
https://arxiv.org/abs/1503.07863
and this one:
https://www.forbes.com/sites/startswithabang/2016/09/09/cosmic-neutrinos-detected-confirming-the-big-bangs-last-great-prediction/?sh=3cd97ae930c7

**Reply by RedshiftDrift** [reply]
All indirect evidence is based on the Big Bang model.  Those neutrinos have not been detected.

Like everything else with the Big Bang model, all evidence is based on the interpretation of redshift as an increasing distance to galaxies and everything that is inferred from that.
